test/cpp/interop/client_helper.cc

```diff
@@ -87,7 +87,12 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
     const std::string& test_case,
     std::vector<
         std::unique_ptr<experimental::ClientInterceptorFactoryInterface>>
-        interceptor_creators) {
+        interceptor_creators,
+    absl::optional<std::string> service_config_json) {
```

Comment:
It would require `ABSL_FLAG` in each client, that would make this PR much bigger. Another option would be to have that declaration in `client_helper.cc` which seems to be discouraged.

Reply:
No longer needed. Clients need to configure all args.